src/controllers/home.js

```diff
@@ -1,4 +1,22 @@
+const renderQueries = require('../queries/queries')
+
+//telling server what to do on the home route
 module.exports = (req, res) => {
-	res.render('home', { activePage: {home: true}});
+
+	//generate random number here, use as id
+	const mentorId1 = Math.floor((Math.random() * 12) + 1);
+	const mentorId2 = Math.floor((Math.random() * 12) + 1);
+	renderQueries.getTwoMentors(mentorId1, mentorId2, (err, res) => {
+		if (err) throw err
```

Comment:
so again in case theres an actual erro this will crash the site cause throw just crashes everything when it runs.
so instead it might be better to render the error page using res